src/process.c

```diff
@@ -287,6 +287,11 @@ cc_run_hook(struct oci_cfg_hook* hook, const gchar* state,
 		return false;
 	}
 
+	if (! hook->path) {
```

Comment:
Wait a sec - having looked more closely here, this change is bogus: `hook->path` is an array, so must be valid if `hook` is. This is alluded to below where we check that the first element is the expected slash to denote a full path.

(I don't see the test you are referring to either. We could do with a new test for the case where `hook->path[0] = '\0'`).